frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -34,16 +34,23 @@ for (const mode of modeList) {
 export default class FeedbackCodeView extends Vue {
   @Prop() language!: string;
   @Prop() value!: string;
-  @Prop({ default: () => new Map<number, typeof FeedbackClass>() })
-  feedbackMap!: Map<number, Feedback>;
+  @Prop({ default: () => new Map<number, ArenaCourseFeedback>() })
+  feedbackMap!: Map<number, ArenaCourseFeedback>;
   @Ref('cm-editor') private readonly cmEditor!: HTMLTextAreaElement;
 
   T = T;
   mode = languageModeMap[this.language] ?? languageModeMap['cpp17-gcc'];
-  mapChangeTracker = 0;
+  mapChangeTracker = 1;
 
-  get mapAsList(): 0 | [number, Feedback][] {
-    return this.mapChangeTracker && Array.from(this.feedbackMap);
+  get mapAsList(): 0 | [number, ArenaCourseFeedback][] {
+    if (!this.mapChangeTracker) {
+      throw new Error('unreachable code');
+    }
+    return Array.from(this.feedbackMap);
+  }
+
+  get numberOfComments(): number {
+    return this.mapAsList != 0 ? this.mapAsList.length : 0;
```

Comment:
```suggestion
    return this.mapAsList.length;
```

esto ya se debería poder así.

Reply:
Listo